x/evm/migrations/v6/migrate.go

```diff
@@ -55,7 +55,7 @@ func MigrateStore(
 		CancunBlock:         paramsV5.ChainConfig.CancunBlock,
 	}
 	params.AllowUnprotectedTxs = paramsV5.AllowUnprotectedTxs
-	params.ActivePrecompiles = paramsV5.ActivePrecompiles
+	params.ActiveStaticPrecompiles = paramsV5.ActivePrecompiles
```

Comment:
nit: will this potentially break if someone reruns the migrations in order? Maybe we don't care about it not sure.